### DISCUSSION TITLE
Moyennes automatiques

### DISCUSSION THREAD

**Discussion by ghost** [a2br/ecoledirecte.js#29] [Q&A] [answered]
Bonjour,
J'aimerai savoir si cette API permet de calculer automatiquement la moyenne EcoleDirecte, même si l'Etablissement ne le permet pas ?

**Comment by a2br** [comment, marked as answer]
Bonjour, 

`ed.js` ne sert qu'à interagir avec l'API EcoleDirecte, et pour l'instant il ne prend pas en charge les opérations mathématiques liées au calcul de la moyenne. C'est une option à envisager dans le futur, et c'est certainement une chose qui sera ajoutée une fois la majeure partie de l'API couverte.

Si tu ne veux l'utiliser que pour calculer ta moyenne, pour le moment je te redirige vers [notes-ed](https://github.com/a2br/notes-ed), un projet Python qui fait ça dans le terminal.
